Classes/GlobalStateExplorers/DatabaseBrowser/FLEXTableContentViewController.m

```diff
@@ -132,7 +132,7 @@ - (void)multiColumnTableView:(FLEXMultiColumnTableView *)tableView didSelectRow:
         
         // Option to delete row
         BOOL hasRowID = self.rows.count && row < self.rows.count;
-        if (hasRowID) {
+        if (hasRowID && self.databaseManager) {
```

Comment:
why do we need this change?

Reply:
To make the option to delete a row only available if databaseManager is not null. It's null if you open the content screen after executing custom query from table screen.

Reply:
It feels that we should figure out why it is `nil` and ensure it's always nonnull? Do you know why it will be `nil` in the scenario you mentioned?

Reply:
It is `nil` when the screen is the result of a custom query instead of tapping on a table.

I think I did this on purpose with the idea in mind that these screens should be immutable? There is a similar check already in `setupToolbarItems` in this file.

But that's silly now that I think about it. Better to just make it `nonnull` like @matrush said

Reply:
That said, `tableName` is `nonnull` and it _should_ be `nullable`… I think I need to do some refactoring here.